internal/backends/mcu/lib.rs

```diff
@@ -32,9 +31,13 @@ mod renderer;
 
 use lengths::*;
 
+/// The Pixel type of the backing store
+/// TODO: make it configurable
+pub type DstPixel = embedded_graphics::pixelcolor::Rgb565;
```

Comment:
I think `DestinationPixel` is a bit long, but to avoid a guess what `Dst` means another option would be `TargetPixel` as an intermediate name. Just a thought, feel free to ignore :-)